test/args_product_test.cc

```diff
@@ -31,6 +31,15 @@ class ArgsProductFixture : public ::benchmark::Fixture {
 
     actualValues.insert(ranges);
   }
+  // Added for completeness - compiler warns about partial override
+  void SetUp(::benchmark::State& state) override {
```

Comment:
but there's a default implementation that calls the const version.. ie, there is no partial override here.  

i don't really want to suggest to users (who use tests as a template) that they need to override both versions.

maybe we need to storm forward into the promised deprecation of the const versions.